src/js/firstparties/twitter.js

```diff
@@ -63,9 +76,33 @@ function unwrapTwitterURLsInTimeline() {
   });
 
   // iterate another time to actually replace the link (of Twitter cards, e.g.)
-  findInAllFrames("a[href^='https://t.co/'], a[href^='http://t.co/'").forEach((link) => {
+  findInAllFrames(tcos_all).forEach((link) => {
+    if (fixes.hasOwnProperty(link.href)) {
+      unwrapTco(link, fixes[link.href]);
+    }
+  });
+}
+
+function unwrapTwitterPwaURLsInTimeline() {
+  // unwrap profile links
+  document.querySelectorAll(tcos_all).forEach((link) => {
+    // use cached value if possible
     if (fixes.hasOwnProperty(link.href)) {
       unwrapTco(link, fixes[link.href]);
+      return;
+    }
+
+    // find span element with link text in it
+    const linkElement = link.getElementsByTagName("span")[0];
```

Comment:
What if there are no `span` elements? Should test to avoid trying to read properties of `undefined` (JS exception).